types/pa-aladdin/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for pa-aladdin 1.0
+// Project: https://github.com/pole1419/pa-aladdin
```

Comment:
this won't work. This should be working URL, existing project, it does not have to be on the GitHub though

Reply:
Moreover, since this is your own package, and few if any members of the public community seem to be using it, you should add these type definitions to your own repo instead of DefinitelyTyped (whether by manually writing a declaration file like this or by simply writing the source in TypeScript).